src/include/libutil.h

```diff
@@ -231,7 +231,8 @@ lock_file(FILE *fp, int op, char *filename, int lock_retry,
 #define RCP_BUFFER_SIZE 65536
 
 
-#define MAXBUFLEN 2048
+#define MAXBUFLEN 1024
+#define BUFFER_GROWTH_RATE 1.5
```

Comment:
Most of the [discussion ](http://stackoverflow.com/questions/1100311/what-is-the-ideal-growth-rate-for-a-dynamically-allocated-array)I found on the web prefers `1.5` over `2` unless it is short lived where you will get the benefit of saving time and not compromising on memory. I've used 1.5 in library functions and utility functions.